Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -232,6 +235,10 @@ public void onSliderClick(SliderSetting item, int position)
     builder.setTitle(item.getNameId());
     builder.setView(view);
     builder.setPositiveButton(R.string.ok, this);
+    builder.setNeutralButton(R.string.default_value, (dialogInterface, i) ->
+    {
+      // Overridden below
+    });
```

Comment:
Does this really have to be done in two steps? It looks to me like you could do it in one step if you put the `setNeutralButton` call after the `SeekBar` code and the `builder.show()` call after the `setNeutralButton` call.

Reply:
Originally I wanted the neutral button to not close the dialog but now that I think about it it would probably be better if it did.

Reply:
Oh, I didn't realize that there was that difference in behavior between the current code and what I suggested. Choose whichever way you want, but if you want to keep the current approach, please insert a comment about why you're doing it this way.

Reply:
I kept my original method and updated the comment. My way may be more useful for AlertDialogs with multiple sliders. Although none of the settings currently use it, I plan on implementing something similar for our in-game AlertDialogs with multiple sliders so I may as well make this consistent.